elasticsearch/utils/set_sg_replicas

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -e
+
+source "logging"
+
+info "Modifying the Searchguard index replica settings."
+
+config=${ES_CONF}/elasticsearch.yml
+index=$(python -c "import yaml; print yaml.load(open('${config}'))['searchguard']['config_index_name']" | xargs -i sh -c "echo {}")
+
+info "Disabling auto replication"
```

Comment:
I think these changes should be made in `utils/es_seed_acl` instead

Reply:
I prefer not.  I find it useful to be able to manually execute seeding.  The current impl allows us to run them separately.

Reply:
Right, shouldn't we always restrict our seeding to not use dynamic replica scaling? we can also default the number of replicas to 0 and allow a variable to configure that. We currently have docs that guide users to run `es_seed_acl` if necessary, they would be in an unfavorable state then.

Reply:
I believe we are confusing ACL seeding vs cluster initialization.  We should always be able to seed the ACLs. I would expect only once to needing to really initialize the cluster and characteristics of the Searchguard index

Reply:
Searchguard indices, once created, have persistent state; that is to say that creating the .searchguard indices are part of cluster initialization.

However, as more ES DCs are created, each new DC will need its cluster initialization step run as well.